evergreen.yml

```diff
@@ -65,17 +65,23 @@ functions:
         shell: bash
         working_dir: mongo-powerbi-connector
         script: |
-          export MONGO_POWERBI_CONNECTOR_VERSION=$(git describe --always | sed 's/^v//g')
           export UPDATE_PLUGIN_VERSION=true
           git describe --exact-match HEAD || UPDATE_PLUGIN_VERSION=false
 
+          if [[ "${triggered_by_git_tag}" != "" ]]; then
+            export release_version=$(echo ${triggered_by_git_tag} | sed s/v//)
+          else
+            export release_version=test_release
+          fi
+
+
           cat <<EOT > expansions.yml
-          MONGO_POWERBI_CONNECTOR_VERSION: "$MONGO_POWERBI_CONNECTOR_VERSION"
+          release_version: "$release_version"
           UPDATE_PLUGIN_VERSION: "$UPDATE_PLUGIN_VERSION"
           prepare_shell: |
             set -o errexit
             export PATH="$PATH"
-            export MONGO_POWERBI_CONNECTOR_VERSION="$MONGO_POWERBI_CONNECTOR_VERSION"
+            export release_version="$release_version"
```

Comment:
nit: How about renaming this to `RELEASE_VERSION` to match the other env variables?